change/@ni-nimble-components-21de2fc8-dfc6-4249-9098-d7842ce0eff3.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Refactored rich text components to use common models",
+  "packageName": "@ni/nimble-components",
+  "email": "123591928+saikrishnan-ni@users.noreply.github.com",
+  "dependentChangeType": "none"
+}
```

Even though this doesn't impact APIs or behavior, it does change the bits of the production package so I think we want to trigger a new version to be published.

```suggestion
{
  "type": "patch",
  "comment": "Refactored rich text components to use common models",
  "packageName": "@ni/nimble-components",
  "email": "123591928+saikrishnan-ni@users.noreply.github.com",
  "dependentChangeType": "patch"
}
```